pyqldbsamples/create_index.py

```diff
@@ -65,8 +65,7 @@ def create_index(transaction_executor, table_name, index_attribute):
                                    and create_index(x, Constants.DRIVERS_LICENSE_TABLE_NAME,
                                                     Constants.PERSON_ID_INDEX_NAME)
                                    and create_index(x, Constants.DRIVERS_LICENSE_TABLE_NAME,
-                                                    Constants.LICENSE_NUMBER_INDEX_NAME),
-                                   lambda retry_attempt: logger.info('Retrying due to OCC conflict...'))
+                                                    Constants.LICENSE_NUMBER_INDEX_NAME))
```

Comment:
Why is this logging gone?

Reply:
In the v2 driver, the driver has a [execute_statement()](https://github.com/awslabs/amazon-qldb-driver-python/blob/v2.x/pyqldb/driver/pooled_qldb_driver.py#L137) function which took in a `retry_indicator` lambda. This lambda would get executed in the event of a retriable exception. 

In v3 driver, we updated this [parameter](https://github.com/awslabs/amazon-qldb-driver-python/blob/master/pyqldb/driver/qldb_driver.py#L203) to take in a `retry_config` object.

It is no longer possible for the user to log a message when there is a retriable exception